src/components/Footer/Footer.jsx

```diff
@@ -1,249 +1,75 @@
-            <h6 className="uppercase font-semibold mb-4 flex justify-center md:justify-start">
-              Pages
-            </h6>
-            <p className="mb-4">
-              <Link to={HOME_ROUTE}>Home</Link>
-            </p>
-            <p className="mb-4">
-              <Link to={ABOUT_ROUTE}>Our Team</Link>
-            </p>
-            <p className="mb-4">
-              <Link to={FAQ_ROUTE}>FAQ</Link>
-            </p>
-          </div>
-          <div className="">
-            <h6 className="uppercase font-semibold mb-4 flex justify-center md:justify-start">
-              Contact
-            </h6>
-            <p className="flex items-center justify-center md:justify-start mb-4">
-              <svg
-                aria-hidden="true"
-                focusable="false"
-                data-prefix="fas"
-                data-icon="home"
-                className="w-4 mr-4"
-                role="img"
-                xmlns="http://www.w3.org/2000/svg"
-                viewBox="0 0 576 512"
-              >
-                <path
-                  fill="currentColor"
-                  d="M280.37 148.26L96 300.11V464a16 16 0 0 0 16 16l112.06-.29a16 16 0 0 0 15.92-16V368a16 16 0 0 1 16-16h64a16 16 0 0 1 16 16v95.64a16 16 0 0 0 16 16.05L464 480a16 16 0 0 0 16-16V300L295.67 148.26a12.19 12.19 0 0 0-15.3 0zM571.6 251.47L488 182.56V44.05a12 12 0 0 0-12-12h-56a12 12 0 0 0-12 12v72.61L318.47 43a48 48 0 0 0-61 0L4.34 251.47a12 12 0 0 0-1.6 16.9l25.5 31A12 12 0 0 0 45.15 301l235.22-193.74a12.19 12.19 0 0 1 15.3 0L530.9 301a12 12 0 0 0 16.9-1.6l25.5-31a12 12 0 0 0-1.7-16.93z"
-                />
-              </svg>
-              New York, NY 10012, US
-            </p>
-            <p className="flex items-center justify-center md:justify-start mb-4">
-              <svg
-                aria-hidden="true"
-                focusable="false"
-                data-prefix="fas"
-                data-icon="envelope"
-                className="w-4 mr-4"
-                role="img"
-                xmlns="http://www.w3.org/2000/svg"
-                viewBox="0 0 512 512"
+    <footer className="p-4 bg-background bg-opacity-50 shadow md:p-5 backdrop-blur-md">
+      <div className="lg:grid lg:grid-cols-4 gap-4">
+        <Link to={HOME_ROUTE}  className="flex lg:flex-row flex-col mb-4 sm:mb-0 col-span-2 lg:justify-start justify-center">
+          <img
+            src="https://cdn.discordapp.com/attachments/1031834305703460906/1035627738440159303/Asset_23.png"
+            className="mr-3 h-32 w-min lg:self-start self-center"
+            alt="Flowbite Logo"
+          />
+          <span className="self-center text-3xl lg:mb-0 mb-3 font-semibold text-primary">
+          Designed to meet your needs.
+          </span>
+          </Link>
+        <div className="col-span-1 flex lg:my-0 my-3 mx-3 lg:justify-start justify-center text-center">
+        <ul className="mb-6 text-lg text-primary sm:mb-0 font-base justify-self-center">
```

Comment:
If you can make links aligned left on large screens and centered on small screens, it would be better.